csaf_2.1/prose/edit/src/schema-elements-02-props-04-vulnerabilities.md

```diff
@@ -639,7 +639,7 @@ Issuing parties SHOULD consider using the SSVC decision point `Provider Urgency`
 an additional assessment provided by a party.
 
 The property SSVC v2 (`ssvc_v2`) holding an SSVC Decision Point Value Selection v2.0.0 value abiding by the schema at
-[https://certcc.github.io/SSVC/data/schema/v2/Decision_Point_Value_Selection-2-0-0.schema.json](https://certcc.github.io/SSVC/data/schema/v2/Decision_Point_Value_Selection-2-0-0.schema.json).
+[https://raw.githubusercontent.com/CERTCC/SSVC/refs/heads/main/data/schema/v2/SelectionList_2_0_0.schema.json](https://raw.githubusercontent.com/CERTCC/SSVC/refs/heads/main/data/schema/v2/SelectionList_2_0_0.schema.json).
```

Comment:
We need to use the value of `$id` (it is just an identifier - programs keep schemas locally). Also, this affects the [design-considerations-01-construction-principles.md](https://github.com/oasis-tcs/csaf/blob/release-csaf-2-1-csd-02-needs-fixes-due-to-eager-merges/csaf_2.1/prose/edit/src/design-considerations-01-construction-principles.md).

I suggest to use the link that seems already to be up:
```suggestion
[https://certcc.github.io/SSVC/data/schema/v2/SelectionList_2_0_0.schema.json](https://certcc.github.io/SSVC/data/schema/v2/SelectionList_2_0_0.schema.json).
```